spec/features/tagging_during_migration_spec.rb

```diff
@@ -0,0 +1,85 @@
+require "rails_helper"
+
+RSpec.describe "Tagging content during migration", type: :feature do
+  include PublishingApiHelper
+
+  before do
+    given_we_can_populate_the_dropdowns_with_content_from_publishing_api
+  end
+
+  scenario "User makes a change to a item without all the link types" do
```

Comment:
"User makes a change to a content item which has had some of its link types disabled"